stream_alert/alert_processor/outputs/github.py

```diff
@@ -0,0 +1,107 @@
+                                        ' Github integration')),
+            ('repository',
+             OutputProperty(description='the repository for this integration '
+                                        'in the form :username/:repository',
+                            cred_requirement=True,
+                            mask_input=False)),
+            ('labels',
+             OutputProperty(description='a comma separated list of labels to '
+                                        'apply to issues when they are created',
+                            cred_requirement=True,
+                            mask_input=False)),
+            ('username',
+             OutputProperty(description='the username for this integration',
+                            cred_requirement=True,
+                            mask_input=False)),
+            ('access_token',
+             OutputProperty(description='the access token for the integration',
+                            cred_requirement=True,
+                            mask_input=True))
+        ])
+
+    @classmethod
+    def _get_default_properties(cls):
+        """Get the standard url used for Github API. This value the same for
+        everyone, so is hard-coded here and does not need to be configured by
+        the user
+
+        Returns:
+            dict: Contains various default items for this output (ie: url)
+        """
+        return {'api': 'https://api.github.com'}
+
+    def dispatch(self, **kwargs):
+        """Send alert to Github
+
+        Args:
+            **kwargs: consists of any combination of the following items:
+                descriptor (str): Service descriptor (ie: slack channel, pd integration)
+                rule_name (str): Name of the triggered rule
+                alert (dict): Alert relevant to the triggered rule
+        """
+        credentials = self._load_creds(kwargs['descriptor'])
+        if not credentials:
+            return self._log_status(False)
+
+        username_password = "{}:{}".format(credentials['username'],
+                                           credentials['access_token'])
+        encoded_credentials = base64.b64encode(username_password)
+        headers = {'Authorization': "Basic {}".format(encoded_credentials)}
+        url = os.path.join(credentials['api'],
+                           "repos/{}/issues".format(credentials['repository']))
+
+        title = "StreamAlert: {}".format(kwargs['rule_name'])
+        body_template = "### Description\n{}\n\n### Event data\n\n```\n{}\n```"
+        body = body_template.format(kwargs['alert']['rule_description'],
+                                    json.dumps(kwargs['alert']['record'], indent=2))
+        issue = {'title': title, 'body': body, 'labels': credentials['labels'].split(',')}
+
+        LOGGER.debug('sending alert to Github repository %s', credentials['repository'])
+
+        success = False
```

Comment:
this declaration is unnecessary since it's defined in both the try and the except blocks